ts/components/screens/LoadingScreenContent.tsx

```diff
@@ -25,8 +25,7 @@ const styles = StyleSheet.create({
     ...IOStyles.flex
   },
   contentTitle: {
-    textAlign: "center",
-    minHeight: 70 // Set minHeight to stabilize title height and avoid shifting
+    textAlign: "center"
```

Comment:
Is it not needed anymore?

Reply:
no because I noticed that the min height on loading leads to side effects 🤯